### PR TITLE
add dockerfile for docker support and change cookie dir

### PR DESCRIPTION
**Pull request by renothing**
change cookie store dir for docker

### CODE REVIEW

File: src/smart_qq_bot/config.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 DEFAULT_PLUGIN_CONFIG = "config/plugin.json"
 
-COOKIE_FILE = 'cookie/cookie.data'
+COOKIE_FILE = '../cookie/cookie.data'
```
Comment (winkidney) [inline src/smart_qq_bot/config.py]: Nice change! It makes the cookie directory a constant:)

But for the path constants, we have two more choices here:
1. the current working directory(If we run run.py, we could get the directory what we expected)
2. An abs directory with "os.path.abspath(os.path.dirname(__file__))"
So "../" is not so fluent as we expected : ), it is the up level of the current working directory.
Could you please change this setting to either of abs paht or working directory?

Comment (winkidney) [inline src/smart_qq_bot/config.py]: BTW, I advise that we put this to change to another commit.

Comment (renothing) [inline src/smart_qq_bot/config.py]: ok. I change it back to origin position.

File: src/smart_qq_bot/http_client.py [hunk 2]
```diff
@@ -24,8 +24,8 @@ class HttpClient(object):
     # urllib2.install_opener(_req)
 
     def __init__(self, load_cookie=False, cookie_file=COOKIE_FILE):
-        if not os.path.isdir("./cookie"):
-            os.mkdir("./cookie")
+        if not os.path.isdir(os.path.dirname(cookie_file)):
```
Comment (winkidney) [inline src/smart_qq_bot/http_client.py]: Have you tested this feature yet?
If tested, we can merge it into the master right now:)

@Yinzo Any other problem?

Comment (Yinzo) [inline src/smart_qq_bot/http_client.py]: I had tested the feather, **all function works great!** 
the http server port is assign by docker, that may need user to use `docker ps` to find that port to use, maybe we can do some improvement on user guide later ;D

May I merge this pull request :D

Comment (winkidney) [inline src/smart_qq_bot/http_client.py]: @Yinzo This a standard review process:)
+ 1
